lib/ops/opsservice/update.go

```diff
@@ -57,7 +57,12 @@ func (o *Operator) RotateSecrets(req ops.RotateSecretsRequest) (*ops.RotatePacka
 		return nil, trace.Wrap(err)
 	}
 
-	resp, err := cluster.rotateSecrets(node, *op)
+	ctx, err := cluster.newOperationContext(*op)
```

Comment:
The label will stick just like, for example, "purpose" and any other label but there's no harm in that.

I also thought about passing the operation instead but decided against it because it will be just another random argument to add to the mess that our "configure" functions accept. I think it's much better to pass operation context everywhere consistently (that's what it's for after all).